tests/integration/search/get.int.test.ts

```diff
@@ -0,0 +1,19 @@
+describe('get function', () => {
+  describe('happy path', () => {
+    it('should find a record', async () => {
+      const response = await fetch('http:/127.0.0.1:3000/v3/technical-records/XYZEP5JYOMM00020/2019-06-15T10:26:53.903Z');
```

Comment:
went with `fetch` since it means we don't actually have to pull in any extra dependencies. Seems to do what we want it to for now anyways. Considered using axios because it does have a slightly nicer API.